### PR TITLE
FIX_multiple_certificate_during_registration_breaks

### PR DESCRIPTION
**Pull request by ioigoume**
Show drop down list if more than one Certificates available.

### CODE REVIEW

File: app/Lib/lang.php [hunk 1]
```diff
@@ -1460,8 +1460,8 @@
   'fd.telephone_number.description' => 'Description',
   // This one is for rendering into a telephone number string (eg: 555 1212 x279)
   'fd.telephone.ext' => 'x',
-  'fd.cert.subject' => 'Subject',
-  'fd.cert.issuer' => 'Issuer',
+  'fd.cert.subject' => 'Cert Subject DN',
```
Comment (NicolasLiampotis) [inline app/Lib/lang.php]: Is there a reason why we need the abbreviation here instead of Certificate?

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Check if we can use full name of Certificate instead of Cert in language definitions; otherwise LGTM

File: app/Lib/lang.php (hunk 1 above)
Comment (ioigoume) [reply app/Lib/lang.php]: Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**